pages/form/Question2.js

```diff
@@ -1,69 +1,79 @@
 import { useState } from "react";
-import {
-  Box,
-  Grid,
-  Heading,
-  Text,
-  Select,
-} from "@chakra-ui/react";
+import { Box, Grid, Heading, Text, Select } from "@chakra-ui/react";
```

Comment:
let's be consistent with our import statements, and keep each imported component on a separate line k?
```suggestion
import {
  Box,
  Grid,
  Heading,
  Text,
  Select,
} from "@chakra-ui/react";
```

Reply:
I have remove the additional line of importing from chakra-ui.